spec/loading_spec.rb

```diff
@@ -35,6 +45,12 @@
         rv = system("ruby -e #{Shellwords.shellescape(code)}")
         expect(rv).to be true
       end
+
+      it 'produces no output' do
```

Comment:
Just to make it clear what we are expecting, I would suggest slight rewording

```suggestion
      it 'produces no warnings in the output'
```

Reply:
I think from the customers' point of view it does not matter if dd-trace-rb produces warnings or some other output, any output that is not meaningful to them shouldn't be produced.